.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Please remove this file. You can put this in your global .gitignore. See: https://help.github.com/articles/ignoring-files/